packages/rum/src/domain/record/observer.ts

```diff
@@ -172,7 +168,7 @@ function initScrollObserver(cb: ScrollCallback, defaultPrivacyLevel: DefaultPriv
       }
       const id = getSerializedNodeId(target)
       if (target === document) {
-        if (isExperimentalFeatureEnabled('visualviewport')) {
+        if (window.visualViewport) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏since this check was not there before, was it a bug or do we not need this check?

Reply:
I added an additional check here since to be safe as I'm not convinced there are zero behavioural affects.

I'm not 100% sure of the various cross browser + cross version affects of:
- `(document.scrollingElement || document.documentElement).scrollLeft` 
vs
- `window.scrollX ?? window.pageXOffset`

**In terms of:**
- inclusion of scrollbar thickness
- measuring visual viewport vs layout
- changes across browser versions

Not a big deal, but I think this should be addressed in a follow up to isolate cause and affect of any replay time rendering differences

Reply:
Got consensus, will change.

Reply:
Got consensus, will change.